test/sqllogictest/cluster.slt

```diff
@@ -41,7 +41,7 @@ statement ok
 CREATE CLUSTER bar REPLICAS (r1 (REMOTE ['localhost:1235']), r2 (REMOTE ['localhost:1236']))
 
 query TT rowsort
-SELECT * FROM mz_clusters
+SELECT CAST(id as NUMERIC), name FROM mz_clusters
```

Comment:
What happens without this?

Reply:
You get an error that looks like this:
> SELECT id, name FROM mz_clusters
    PlanFailure:test/sqllogictest/id_reuse.slt:103:
    error deserializing column 2: a Postgres value was `NULL`: a Postgres value was `NULL`

I'm honestly think I might just port the stuff from #14401 so I can get rid of this.